terraform/memorystore/modules/test/memorystore.tf

```diff
@@ -0,0 +1,13 @@
+module "memorystore" {
+  source  = "terraform-google-modules/memorystore/google"
+  version = "~> 4.4.0"
+
+  name    = "${var.prefix}-memorystore"
+  project = var.project_id
+
+  region       = var.region
+  tier         = "BASIC"
+  auth_enabled = true
+
+  labels = var.labels
```

Comment:
Looks good--pretty neat that we can use a module. Does this create an instance of the minimum 1GB size by default? 

The other thing is that we will probably want to force API enablement, similar to what I did here: https://github.com/cloudquery/cq-provider-gcp/pull/415

Reply:
1 - yes, the [default](https://github.com/terraform-google-modules/terraform-google-memorystore/blob/master/variables.tf#L54) is 1 GiB
2 - added the API enablement